docs/design/keyboard-shortcuts.md

```diff
@@ -0,0 +1,184 @@
+
+1. Be sure there is an outermost pair of braces (`{ }`)in the file.
+1. Just inside this outermost object, add the following JSON markup. Note that the file must be proper JSON, not simple a JavaScript object, so the property names must be within quotation marks.
+
+    ```javascript
+    {
+        "shortcuts": [
+        ]
+    }
+    ```
+
+1. The shortcuts array will contain objects that map key combinations onto action names. Here is an example. Note the following about this markup:
+
+    - The property names you see here, `action`, `key`, and `default` are mandatory.
+    - The value of the `action` property can be any string, and the `default` property can be any combination of the characters A - Z, a -z, 0 - 9, and the punctuation marks "-", "_", and "+". (By convention lower case letters are not used in these properties.)
+    - The `default` property must contain the name of at least one modifier key (ALT, CTRL, SHIFT) and at least one other key. (An additional modifier key is possible on Mac. See the next step.)
+    - When two characters are linked to the same physical key in a standard keyboard, then they are synonyms in the `default` property; for example, ALT+a and ALT+A are the same shortcut, so are CTRL+- and CTRL+\_ because "-" and "_" are the same physical key.
+    - The "+" character indicates that the keys on either side of it are pressed simultaneously.
+    - In a later step, the action names will themselves be mapped to functions that you write. In this example, you will later map SHOWTASKPANE to a function that calls the `Office.addin.showAsTaskpane` method and HIDETASKPANE to a function that calls the `Office.addin.hide` method. 
+
+    > [!NOTE]
+    > The complete schema for the shortcuts JSON is at [extended-manifest.schema.json](https://developer.microsoft.com/en-us/json-schemas/office-js/extended-manifest.schema.json).
+
+    ```javascript
+    {
+        "shortcuts": [
+            {
+                "action": "SHOWTASKPANE",
+                "key": {
+                    "default": "CTRL+SHIFT+UP"
+                }
+            },
+            {
+                "action": "HIDETASKPANE",
+                "key": {
+                    "default": "CTRL+SHIFT+DOWN"
+                }
+            }
+        ]
+    }
+    ```
+
+    > [!NOTE]
+    > Keytips, also known as sequential key shortcuts, such as the Excel shortcut to choose a fill color **Alt+H, H**, are not supported in Office add-ins.
+
+1. Optionally, you can vary the key combination for Office on the web, Office on Windows, or Office on Mac with additional properties on the `"key"` property. The following is an example. The `"default"` combination is used on any platform that doesn't have it's own specified combination. Note that on Mac, you can use the modifier key COMMAND.
```

Comment:
OPTION key is also possible on Mac. Note that the modifiers are NOT case-sensitive.